tests/mitmproxy_addons/callback.py

```diff
@@ -32,38 +47,64 @@ def __init__(self):
 
     def reset(self):
         self.config = {
-            "callback_url": "",
+            "callback_request_url": "",
+            "callback_response_url": "",
```

Comment:
given that the whole thing is a Callback, the `callback_` prefix for these names seems a bit redundant? YMMV

Reply:
I prefix this on purpose because there are a lot of request/response keys flying around (the ones you register with the addon, the ones passed in as req body to the callback server, the res body from the callback server) and I've tried to use `callback_` for configuring the callback server, `respond_` for the response body from the callback server and no special prefix for the callback data itself.